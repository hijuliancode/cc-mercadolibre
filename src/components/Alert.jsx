import React from 'react'

export const Alert = ({ children }) => {
  console.log('Alert.jsx')
  return (
    <div className="alert">
      {children}
    </div>
  )
}
