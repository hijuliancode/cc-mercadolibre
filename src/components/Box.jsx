import React from 'react'

export const Box = ({ children }) => {
  console.log('Box.jsx')
  return (
    <div className="box">
      {children}
    </div>
  )
}
