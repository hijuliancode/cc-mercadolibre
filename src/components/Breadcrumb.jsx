import React from 'react'

export const Breadcrumb = ({ items }) => {
  return (
    <ul className="breadcrumb">
      <li className="breadcrumb__item">item</li>
    </ul>
  )
}
