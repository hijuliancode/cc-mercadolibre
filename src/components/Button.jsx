import React from 'react'

export const Button = ({ value, onHandleClick }) => {
  return (
    <button className="container" onClick={() => onHandleClick}>
      {value}
    </button>
  )
}
