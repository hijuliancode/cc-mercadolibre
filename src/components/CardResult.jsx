import React from 'react'

export const CardResult = ({ data }) => {
  return (
    <div className="card-result">
    </div>
  )
}
