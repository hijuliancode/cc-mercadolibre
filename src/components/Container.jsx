import React from 'react'

export const Container = ({ className = '', children }) => {
  console.log('Container.jsx')
  return (
    <div className={`container ${className}`.trim()}>
      {children}
    </div>
  )
}
