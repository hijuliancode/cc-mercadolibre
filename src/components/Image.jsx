import React from 'react'

export const Image = ({ src, alt }) => {
  return (
    <figure className="image">
      <img src={src} alt={alt} loading="lazy" />
    </figure>
  )
}
