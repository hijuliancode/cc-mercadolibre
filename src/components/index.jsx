export * from './Alert'
export * from './Box'
export * from './Breadcrumb'
export * from './Button'
export * from './CardResult'
export * from './Container'
export * from './Image'
export * from './SearchBar'
export * from './Text'
export * from './shared/DemoComponent'
export * from './shared/Footer'
export * from './shared/Header'
