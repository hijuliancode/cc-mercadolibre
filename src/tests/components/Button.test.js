import React from 'react'
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import { Button } from '../../components/Button';


describe('test in Button.jsx', () => {
  let wrapper;
  
  test('should render <Button />', () => {
    wrapper = shallow( <Button/> )
    expect( wrapper ).toMatchSnapshot()
  })
  
});
