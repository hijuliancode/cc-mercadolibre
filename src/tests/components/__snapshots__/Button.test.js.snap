// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test in Button.jsx should render <Button /> 1`] = `
<button
  className="container"
  onClick={[Function]}
/>
`;
